apps/marketing/src/app/not-found.tsx

```diff
@@ -0,0 +1,66 @@
+'use client';
+
+import Image from 'next/image';
+import { useRouter } from 'next/navigation';
+
+import { motion } from 'framer-motion';
+import { ChevronLeft } from 'lucide-react';
+
+import { cn } from '@documenso/ui/lib/utils';
+import { Button } from '@documenso/ui/primitives/button';
+
+import backgroundPattern from '~/assets/background-pattern.png';
+
+export default function NotFound() {
+  const router = useRouter();
+  return (
+    <motion.div className={cn('relative')}>
+      <div className="absolute -inset-24 -z-10">
+        <motion.div
+          className="flex h-full w-full origin-top-right items-center justify-center"
+          initial="initial"
+          animate="animate"
```

Comment:
We haven't passed in any variants so these two wouldn't be doing anything right now 🤔

Reply:
Updated on my end, cheers!